ssssort.h

```diff
@@ -246,9 +245,8 @@ struct Classifier {
      * classify_unroll before to fill the bktout and bktsize arrays.
      */
     template <std::size_t U>
-    inline void
-    distribute(InputIterator in_begin, InputIterator in_end,
-               OutputIterator out_begin)
+    void distribute(InputIterator in_begin, InputIterator in_end,
```

Comment:
Exactly that: it's ignored by modern compilers. Nowadays `inline` is only useful to avoid ODR problems. Since the function is already a template, you're guaranteed that the compiler always sees its code when it uses it, so it can perform inlining and aggressive optimizations as needed.

You can add it back if you're unsure :p